Update Developer Guide on Codecov for PR Review (#2659)
MarkBind/markbind: docs/devGuide/projectManagement.md

  This is to allow other developers the chance to review the PR, and delay the merge if there's any significant problems. Not everyone is available all of the time, so please be patient.
  </box>

  <box type="info" seamless>
  

>   As part of the PR review process, use the Codecov report generated by the CI pipeline to assess whether the proposed changes significantly degrade test coverage. Depending on the context, you may or may not opt to do anything about this degradation.

Reviewer: ```suggestion
  As part of the PR review process, use the Codecov report generated by the CI pipeline to assess whether the proposed changes significantly degrade test coverage. Depending on the severity of the degradation, you may have to check the cause and address it accordingly.
```

>   Note that code coverage degradations will be result in a **false-positive CI failure** on the `master` branch.

Reviewer: ```suggestion
  Note that code coverage degradation may result in a **false-positive CI failure** on the `master` branch.
```
